### DISCUSSION TITLE
adding javascript

### DISCUSSION THREAD

**Discussion by seanchien4** [daattali/beautiful-jekyll#1056] [Q&A]
I have a html page which i want to add a filter so i need to use javascript. I wonder how I can add the javascript to a specific page. 
for ex: my page name is test.html and it is under the navi bar and it shows normally
I saw there is a line in _config.yml that allow me to add CSS and javascript. 
and it says 
side-js:
    -  "/assets/js/side.js"

in my case, should I do 
test-js:
    -  "/assets/js/test.js"

Thanks!

**Comment by daattali** [comment]
Anything in the config file applies to the entire site. If you want to affect only one page, then you use the YAML front matter. You would use either `js` or `ext-js`, you can see it in the list of parameters here https://github.com/daattali/beautiful-jekyll#advanced-parameters
